### DISCUSSION TITLE
To integrate one-app Enterprise level

### DISCUSSION THREAD

**Discussion by iyyoo93** [americanexpress/one-app#633] [Q&A] [answered]
I  worked as a contractor  in American Express as frontend engineer using One-App framework.
In my current organisation we are planning for web page migration and doing analysis on different approaches to pick the best.
 
As I have experience with One-App(Hope its open source now) I have suggested it and see One-App may be right choice for us to pick. But our major concern is One-App is not yet globally picked up and cant see much discussion your One-App github community. So if we struck somewhere after picking it or struggle while setting up from scratch in terms of deployments, SSR, CSR, or some tech stack, Shall we expect any prompt support form your team when we approach ?.
 
Also we would like know what kind of support you currently have to reach for community discussions or to resolve bottle necks. Please suggest us do we need to keep anything in mind before picking one-app based on your One-App experience.

Happy to discuss over call regarding our high level idea of migration.

**Comment by 10xLaCroixDrinker** [comment, marked as answer]
Hey @iyyoo93,we keep an eye out for any community support that may be needed to keep on top of it, but I can't promise any SLA. It is a small team, so we can't offer a whole lot unfortunately. I'd be happy to set up a call for January to discuss your use case. Please email me to do so at jamie.king@aexp.com. Let me know in the message what time zone you're in, what your migration timeline looks like and who all should be invited as well as any other details you think may be helpful.

**Reply by iyyoo93** [reply]
@10xLaCroixDrinker  Have sent out the mail with more infos.

**Comment by mansisce** [comment]
@iyyoo93 did you adopt one-app for your project?
